src/layouts/Footer/FooterItem.spec.tsx

```diff
@@ -28,9 +28,14 @@ describe('FooterItem', () => {
       const a = screen.getByRole('link')
       expect(a).toBeInTheDocument()
     })
+
+    it('is the right color', () => {
```

Comment:
we don't test for color too frequently, but I like it. I may steal this 😝

Reply:
Lol yeah I asked @calvin-codecov too as it seemed overboard but it seemed like a lot of things were yelling at me about lowered coverage so thought I'd appease it for my first commit... I'll let you know when I form a better opinion on it, lmk if you do too 😂